velox/core/PlanNode.h

```diff
@@ -885,6 +885,14 @@ class PartitionFunctionSpec : public ISerializable {
 
 using PartitionFunctionSpecPtr = std::shared_ptr<const PartitionFunctionSpec>;
 
+// Stores partitioned output channels.
+// For each 'kConstantChannel', there is an entry in 'constValues'.
+struct PartitionedOutputChannels {
+  std::vector<column_index_t> channels;
+  // Each vector holding a single value for a constant channel.
+  std::vector<VectorPtr> constValues;
+};
+
```

Comment:
Can you expose deserializePlanNodeId so we can deprecate a couple of implementations in presto cpp.

Reply:
Let's do it in another PR to keep this one clean?